### ISSUE TITLE
Can you support Packer?

### ISSUE THREAD

**Issue by tsoernes** [r-darwish/topgrade#974]
### Which tool is this about? Where is its repository?
neovim plugin manager
https://github.com/wbthomason/packer.nvim
### Which exact commands should Topgrade run?
`:PackerSync`
## I want to suggest some general feature
Topgrade should...

**Comment by mrjones2014** [comment]
> Which exact commands should Topgrade run?

More accurately, to update Packer plugins from the command line, you can do:

```shell
nvim --headless -c 'autocmd User PackerComplete quitall' -c 'PackerSync'
```

This will:

- Open Neovim headless
- Set up an autocmd to quit Neovim when Packer is done
- Run `:PackerSync` in Neovim

**Comment by svenXY** [comment]
did you see #756 ? packer.nvim is actually already supported, but does not correctly work

**Comment by mrjones2014** [comment]
> due to the asynchronous way packer is run

Yeah, the command I posted above solves this -- it opens Neovim headless, sets up an autocmd, then runs packer, so it waits for packer to finish before exiting neovim.

```
nvim --headless -c 'autocmd User PackerComplete quitall' -c 'PackerSync'
```

**Comment by wbthomason** [comment]
The PR also died because I didn't have time to keep debugging it at the time. It was unclear if the change I proposed to the arguments `topgrade` passes to Neovim actually solved the issue or merely revealed another. If you'd like to pick the PR up where I left it, it's still the basic approach I would take.

A reasonable alternative may be to define a custom command for `topgrade` with the command @mrjones2014 suggests above.
